### PR TITLE
sys-block/zram-init: version bump

### PR DESCRIPTION
**Pull request by beatussum**

### CODE REVIEW

**Comment by beatussum** [comment]
@perfinion ping

**Comment by mjeveritt** [comment]
This can be bumped again up to 10.3.

Ebuild looks good to me - you may want to [open a bug](https://bugs.gentoo.org) if Jason isn't following here. I've pinged him on IRC and will be following this up for my own use :smiley: !

**Comment by beatussum** [comment]
@mjeveritt As requested, I've bumped it up to **10.3**, and I'm going to open a bug. Thanks for your reply! :+1:

File: sys-block/zram-init/zram-init-10.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+to the boot runlevel:
+  rc-update add zram-init boot
+Still for the same case,    you should add in the OpenRC configuration file
+for the services using \$TMPDIR the following line:
+  rc_need=\"zram-init\""
+
+src_prepare() {
+	default
+
+	hprefixify "${S}/man/${PN}.8"
```
Comment (beatussum) [inline sys-block/zram-init/zram-init-10.5.ebuild:49]: I'm not sure about that: these lines are just documentation fields `hprefixify`ed.

File: sys-block/zram-init/zram-init-10.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+	hprefixify -e "s%(}|:)(/(usr/)?sbin)%\1${EPREFIX}\2%g" \
+		"${S}/sbin/${PN}.in"
+
+	hprefixify -e "s%( |=)(/tmp)%\1${EPREFIX}\2%g" \
+		"${S}/systemd/system"/* \
+		"${S}/openrc"/*/*
```
Comment (beatussum) [inline sys-block/zram-init/zram-init-10.5.ebuild:56]: same thing.

**Comment by beatussum** [comment]
@perfinion ping

**Pull request closed without merging**